ocs_ci/helpers/cnv_helpers.py

```diff
@@ -328,3 +335,53 @@ def get_ssh_private_key_path():
     )
 
     return private_key_path
+
+
+def cal_md5sum_vm(vm_obj, file_name, username=None):
+    """
+    Calculate the MD5 checksum of a file via SSH on a virtual machine.
+
+    Args:
+        vm_obj (obj): The virtual machine object.
+        file_name (str): The name of the file to calculate the MD5 checksum for.
+        username (str, optional): The username to use for SSH authentication. Defaults to None.
+
+    Returns:
+        str: The MD5 checksum of the specified file.
+
+    """
+    md5sum_out = vm_obj.run_ssh_cmd(
+        command=f"md5sum {file_name}",
+        username=username,
+    )
+    return md5sum_out.split()[0]
+
+
+def run_dd_io(
+    vm_obj, file_name, bs="1024", count="102400", username=None, verify=False
```

Comment:
Do we really need both bs and count arguments? Shouldn't we simplify it by just asking the user for the file size this function has to create?

Reply:
Made bs as default to 1MB and count(aka size) is size_in_megabytes.